examples/arkode/CXX_serial/CMakeLists.txt

```diff
@@ -68,7 +68,7 @@ foreach(example_tuple ${ARKODE_examples})
 
     set_target_properties(${example_target} PROPERTIES FOLDER "Examples")
 
-    target_compile_features(${example_target} PRIVATE cxx_std_11)
+    target_compile_features(${example_target} PRIVATE cxx_std_14)
```

Comment:
I think these could actually be removed, I believe `CMAKE_CXX_STANDARD` overrides this

Reply:
They could be I think, but not sure what other effects it would have so didnt do it in this PR (I only updated them as it was causing an error).

Reply:
Sounds good, we can leave them for now and see about removing them after the release.